docs/sources/operators-guide/reference-learning-resources/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Reference: Learning resources"
+menuTitle: "Learning resources"
```

Comment:
```suggestion
menuTitle: "Reference: Learning resources"
```
```suggestion
menuTitle: "Learning resources"
```

Reply:
Current version is:

```
menuTitle: "Learning resources"
```

Should keep it or update to `menuTitle: "Reference: Learning resources"` ? (I see two suggestions)